roc/receiver.go

```diff
@@ -0,0 +1,37 @@
+package roc
+
+/*
+#cgo LDFLAGS: -lroc
+#include <roc/receiver.h>
+#include <roc/address.h>
+#include <roc/sender.h>
+#include <roc/log.h>
+#include <stdlib.h>
+*/
+import "C"
+
+import (
+	"unsafe"
+)
+
+func OpenReceiver(rocContext *Context, receiverConfig *ReceiverConfig) (*Receiver, error) {
+	rocContextCPtr := (*C.roc_context)(unsafe.Pointer(rocContext))
```

Comment:
Can we replace `(*C.roc_context)(unsafe.Pointer(rocContext))` with just `(*C.roc_context)(rocContext)`? And maybe get rid of the var at all.

Reply:
done